tools/CouchDB_MRI_Importer.php

```diff
@@ -88,14 +88,19 @@ function _generateCandidatesQuery($ScanTypes)
 
         foreach($s as $scan){
             $scantype=$scan['ScanType'];
-            $Query .= ", (SELECT f.File FROM files f LEFT JOIN mri_scan_type msc
-              ON (msc.ID= f.AcquisitionProtocolID)
-              WHERE f.SessionID=s.ID AND msc.Scan_type='$scantype' LIMIT 1)
-                    as Selected_$scantype, (SELECT fqc.QCStatus
-                    FROM files f 
+            $Query  .= ", (SELECT f.File FROM files f
+                    LEFT JOIN files_qcstatus fqc USING(FileID)
+                    LEFT JOIN mri_scan_type msc ON (msc.ID= f.AcquisitionProtocolID)
+              WHERE f.SessionID=s.ID AND msc.Scan_type='$scantype'
+              AND fqc.Selected='true' ORDER BY
+              FIND_IN_SET(fqc.QCStatus, 'Fail,Pass') DESC LIMIT 1)
+                    as Selected_$scantype,
+              (SELECT fqc.QCStatus FROM files f
                     LEFT JOIN files_qcstatus fqc USING(FileID)
                     LEFT JOIN mri_scan_type msc ON(msc.ID= f.AcquisitionProtocolID)
-              WHERE f.SessionID=s.ID AND msc.Scan_type='$scantype' LIMIT 1)
+              WHERE f.SessionID=s.ID AND msc.Scan_type='$scantype'
+              AND fqc.Selected='true' ORDER BY
+              FIND_IN_SET(fqc.QCStatus, 'Fail,Pass') DESC LIMIT 1)
```

Comment:
![findinset](https://user-images.githubusercontent.com/2903505/28781756-20295d4e-75d9-11e7-8104-a92f26257219.png)

@xlecours  FIND_IN_SET returns 2 when it's a pass, that makes this sort work.
Studies if there is at least one scan from this protocol that was a pass, then if it was a fail, then if it was selected yet not qced.